agentassist/src/app/pages/feedback-settings/feedback-settings.component.ts

```diff
@@ -308,6 +309,7 @@ export class FeedbackSettingsComponent implements OnInit {
   }
 
   closeDropdown(){
+    this.suggestionTitle = '';
     this.suggestionDropdowns.forEach(dropdown => {
       dropdown.close();
```

Comment:
@saradadh-Kore will the dropdownalways available.
if not please add null check.

Reply:
updated check once